src/app/pages/send-crypto-asset/hooks/use-fees.ts

```diff
@@ -0,0 +1,49 @@
+import { useMemo } from 'react';
+
+import { useBitcoinFees } from '@app/query/bitcoin/fees/fee-estimates.hooks';
+import { useStacksFees } from '@app/query/stacks/fees/fees.hooks';
+import {
+  useFtTokenTransferUnsignedTx,
+  useStxTokenTransferUnsignedTxState,
+} from '@app/store/transactions/token-transfer.hooks';
+
+import {
+  getEstimatedUnsignedStacksTxByteLength,
+  getSerializedUnsignedStacksTxPayload,
+} from '../form-utils';
+import { useStacksFtParams } from '../forms/stx-sip10/use-stacks-ft-params';
+
+export function useBtcFees() {
+  // TODO: Replace hardcoded number here (200) with the tx byte length
+  const { data: btcFees } = useBitcoinFees(200);
+  return btcFees;
+}
+
+export function useStxFees() {
+  const unsignedTx = useStxTokenTransferUnsignedTxState();
+
+  const { txByteLength, txPayload } = useMemo(() => {
+    return {
+      txByteLength: getEstimatedUnsignedStacksTxByteLength(unsignedTx),
+      txPayload: getSerializedUnsignedStacksTxPayload(unsignedTx),
+    };
+  }, [unsignedTx]);
+
+  const { data: stxFees } = useStacksFees(txByteLength, txPayload);
+  return stxFees;
+}
```

Comment:
Same here. I see the complexity regarding the fee estimation/txbytes, but sure there's a way around this. Happy to chat through ideas how we can reorganise this

Reply:
I will redo, np.